rpc/rpcserver/server.go

```diff
@@ -682,6 +682,69 @@ func (s *walletServer) StakeInfo(ctx context.Context, req *pb.StakeInfoRequest)
 	}, nil
 }
 
+// makeChangeSourceFromAddress creates a ChangeSource which is used to
+// receive all correlated previous input value.
+func makeChangeSourceFromAddress(address string) txauthor.ChangeSource {
+	return func() ([]byte, uint16, error) {
+		destinationAddress, err := dcrutil.DecodeAddress(address)
+		if err != nil {
+			return nil, 0, err
+		}
+		script, err := txscript.PayToAddrScript(destinationAddress)
+		if err != nil {
+			return nil, 0, err
+		}
+
+		return script, txscript.DefaultScriptVersion, err
+	}
+}
+
+func (s *walletServer) SweepAccount(ctx context.Context, req *pb.SweepAccountRequest) (*pb.SweepAccountResponse, error) {
+	// use provided fee per Kb if specified.
+	feePerKb := s.wallet.RelayFee()
+	if req.FeePerKb > 0 {
+		var err error
+		feePerKb, err = dcrutil.NewAmount(req.FeePerKb)
+		if err != nil {
+			return nil, err
```

Comment:
unless this is using one of the unexported helper functions from this package which already returns a grpc error, use `translateError(err)`, or an explicit `status.Errorf(codes.InvalidArgument, "%v", err)` in this case.  The entire RPC handler should do this to avoid unnecessary `codes.Unknown` errors.